Libplanet/Net/Swarm.cs

```diff
@@ -1337,18 +1340,37 @@ private void TransferTxs<T>(BlockChain<T> blockChain, GetTxs getTxs)
             where T : IAction, new()
         {
             _logger.Debug("Trying to transfer blocks...");
+
+            var blocks = new List<byte[]>();
+
             foreach (HashDigest<SHA256> hash in getData.BlockHashes)
             {
                 if (blockChain.Blocks.TryGetValue(hash, out Block<T> block))
                 {
-                    Message response = new Block(block.ToBencodex(true, true))
+                    var payload = block.ToBencodex(true, true);
+                    blocks.Add(payload);
+                }
+
+                if (blocks.Count == getData.ChunkSize)
+                {
+                    Message response = new Messages.Blocks(blocks.ToArray())
```

Comment:
Should we re-materialize `blocks` with `.ToArray()`? `blocks` seems to be already non-lazy enumerable.

Reply:
You're right. However, if I call Clean after putting the `blocks` in a queue, the messages in the queue are affected. So I called ToArray to copy it.

Reply:
Ok, I see. on the other hand, it seems that copying from the `Blocks` constructor is safer than the caller's side if copy is needed.